pyoverkiz/enums/state.py

```diff
@@ -124,12 +127,15 @@ class OverkizState(str, Enum):
     CORE_WATER_DETECTION = "core:WaterDetectionState"
     CORE_WEATHER_STATUS = "core:WeatherStatusState"
     CORE_WIND_SPEED = "core:WindSpeedState"
+    FREEZE_MODE = "freezeMode"
+    FURTHER_NOTICE = "further_notice"
     HLRRWIFI_FAN_SPEED = "hlrrwifi:FanSpeedState"
     HLRRWIFI_LEAVE_HOME = "hlrrwifi:LeaveHomeState"
     HLRRWIFI_MAIN_OPERATION = "hlrrwifi:MainOperationState"
     HLRRWIFI_MODE_CHANGE = "hlrrwifi:ModeChangeState"
     HLRRWIFI_ROOM_TEMPERATURE = "hlrrwifi:RoomTemperatureState"
     HLRRWIFI_SWING = "hlrrwifi:SwingState"
+    INACTIVE = "inactive"
```

Comment:
Is this a state or a state param?

Reply:
This is a state value, sorry. We don’t use enum for state value, right?

Reply:
I just checked, inactive and active are never used as command parameter, they are just a state value.

Reply:
In theory we could still add it, so you can use the enum, however than the name OverkizCommandParam is a bit weird.